test/run.py

```diff
@@ -1618,6 +1618,15 @@ def test_yield_sequences_in_list(self):
         ])
         self.assertEquals(actual, expected)
 
+    def testIgnoreFrameSetStrings(self):
+        for char in "xy:,".split():
```

Comment:
@walrusVision, I just noticed that this test loop isn't doing what I think you intended. If it was supposed to test each character, it is actually only testing a single "xy:," because the string `split()` operation defaults to whitespace. To just test each character, you would omit the `split()` and python will loop over each char. 

Fortunately I tested this locally and the tests still pass. If its not what you intended here, we should correct it.